versioned_docs/version-0.14/configuration/index.md

```diff
@@ -0,0 +1,206 @@
+```
+
+The following properties need such `null` value during upgrade, if you want to keep the previous configuration:
+
+- `communications.default-group.{communication-platform}.channels.default.notifications`, where `{communication-platform}` is any communication platform supported except Microsoft Teams,
+- `communications.default-group.{communication-platform}.channels.default.bindings.sources`, where `{communication-platform}` is any communication platform supported except Microsoft Teams,
+- `communications.default-group.teams.bindings.sources`.
+
+To learn more, read the [Deleting a default key](https://helm.sh/docs/chart_template_guide/values_files/#deleting-a-default-key) paragraph in Helm documentation.
+
+## Environment variables
+
+The individual communication settings can be specified via environment variables. They take priority and override the configuration specified in the file.
+
+To construct the environment variable name, take any property from the configuration file and make it uppercase. Use the underscore for properties that are nested. Use the double underscore for all camelCase properties. Finally, add the `BOTKUBE_` prefix.
+
+For example, such configuration property from YAML:
+
+```yaml
+settings:
+  kubectl:
+    defaultNamespace: "NAMESPACE"
+```
+
+is mapped to the `BOTKUBE_SETTINGS_KUBECTL_DEFAULT__NAMESPACE` environment variable.
+
+This is a useful feature that allows you to store the overall configuration in a file, where sensitive data, such as tokens, can be put in environment variables. See the [**Tokens from Vault via CSI driver**](./communication/vault-csi/) tutorial for an example use-case.
+
+## Merging strategy
+
+BotKube allows you to split individual settings into multiple configuration files. The following rules apply:
+
+- The priority will be given to the last (right-most) file specified.
+- Files with `_` name prefix are always read as the last ones following the initial order.
+- Objects are merged together and primitive fields are overridden. For example:
+
+  ```yaml
+  # a.yaml - first file
+  settings:
+    clusterName: dev-cluster
+    configWatcher: true
+    kubectl:
+      enabled: false
+  ```
+
+  ```yaml
+  # _a.yaml - second file with `_` prefix
+  settings:
+    clusterName: demo-cluster
+  ```
+
+  ```yaml
+  # b.yaml - third file
+  settings:
+    kubectl:
+      enabled: true
+  ```
+
+  ```yaml
+  # result
+  settings:
+    clusterName: demo-cluster
+    configWatcher: true
+    kubectl:
+      enabled: true
+  ```
+
+- The arrays items are not merged, they are overridden. For example:
+
+  ```yaml
+  # a.yaml
+  settings:
+    kubectl:
+      enabled: true
+      commands:
+        verbs:
+          ["api-resources", "api-versions", "cluster-info", "describe", "diff", "explain", "get", "logs", "top", "auth"]
+  ```
+
+  ```yaml
+  # b.yaml
+  settings:
+    kubectl:
+      commands:
+        verbs: ["get", "logs", "top", "auth"]
+  ```
+
+  ```yaml
+  # b.yaml
```

Comment:
Please apply the change from https://github.com/kubeshop/botkube-docs/commit/6dcf3aa415d3f3c01bd43f090159b5fb76cf27d6:

```suggestion
  # result
```